Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -183,7 +184,11 @@ class ASHorizontalScrollCell: UITableViewCell {
         let pageControl = FilledPageControl()
         pageControl.tintColor = UIColor.grayColor()
         pageControl.indicatorRadius = ASHorizontalScrollCellUX.PageControlRadius
-        pageControl.userInteractionEnabled = false
+        pageControl.userInteractionEnabled = true
+        pageControl.isAccessibilityElement = true
+        pageControl.accessibilityIdentifier = "pageControl"
+        pageControl.accessibilityLabel = NSLocalizedString("TopSites Page control", comment: "The page control button that lets you switch between pages.")
```

Comment:
I would put this in the Top Sites section of `Strings.swift` [1] and follow the same format as the other strings defined there.
[1] https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Strings.swift#L34